_data-prepper/pipelines/configuration/sources/dynamo-db.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: default
+title: dynamodb
+parent: Sources
+grand_parent: Pipelines
+nav_order: 3
+---
+
+# dynamodb
+
+The `dynamodb` source enables change data capture (CDC) which streams [DynamoDB](https://aws.amazon.com/dynamodb/) table events such as `create`, `update`, or `delete` to Data Prepper events in near real-time.
```

Comment:
```suggestion
The `dynamodb` source enables change data capture (CDC) on [DynamoDB](https://aws.amazon.com/dynamodb/) tables. It can receive table events such as `create`, `update`, or `delete` via DynamoDB streams and supports initial snapshots using [point-in-time recovery (PITR)](https://aws.amazon.com/dynamodb/pitr/).
```